src/types/index.ts

```diff
@@ -131,6 +131,16 @@ export type CloneOptions =
 			enhance?: boolean;
 	  };
 
+export type HifiCloneOptions = {
+	clone_options: CloneOptions;
+	name: string;
+	description: string;
```

Comment:
shouldn't description be optional?

Reply:
I want it to be empty string instead of null